lib/offline/storage.js

```diff
@@ -770,6 +788,61 @@ shaka.offline.Storage.prototype.createOfflineManifest_ = function(
   };
 };
 
+/**
+ * Extract DRM initData from first segment
+ *
+ * @param {shaka.extern.Manifest} manifest
+ * @param {shaka.net.NetworkingEngine} networkingEngine
+ * @return {Promise<shaka.extern.Manifest>}
+ * @private
+ */
+shaka.offline.Storage.prototype.extractInitDataFromFirstSegment_ = async function(
```

Comment:
Is there a reason why this is only needed to be done with stored content? Why would we never need to do this for streaming content?

Reply:
Because with streaming content the browser fires `encrypted` events when it sees a `pssh` box, so we don't need to parse it.

Reply:
this is indeed because for online playback EME is handling the pssh extraction.